clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -205,13 +205,15 @@ class CIRBaseBuilderTy : public mlir::OpBuilder {
   // Call operators
   //===--------------------------------------------------------------------===//
 
-  cir::CallOp createCallOp(mlir::Location loc, mlir::SymbolRefAttr callee) {
-    auto op = create<cir::CallOp>(loc, callee);
+  cir::CallOp createCallOp(mlir::Location loc, mlir::SymbolRefAttr callee,
+                           mlir::Type returnType = cir::VoidType()) {
+    auto op = create<cir::CallOp>(loc, callee, /*resType=*/returnType);
```

Comment:
```suggestion
    auto op = create<cir::CallOp>(loc, callee, returnType);
```

We typically don't do these comments unless they make the code more clear/aren't obvious from the variable.  So typically only for arguments that are literals.